blog/models.py

```diff
@@ -0,0 +1,41 @@
+from django.conf import settings
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class User(AbstractUser):
+    username = models.CharField(max_length=255, unique=True)
+    email = models.EmailField(unique=True)
+    first_name = models.CharField(max_length=255)
+    last_name = models.CharField(max_length=255)
+    password = models.CharField(max_length=255)
+
+    class Meta:
+        ordering = ("username",)
+
+
+class Post(models.Model):
+    owner = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE,
+        related_name="posts"
+    )
+    title = models.CharField(max_length=255)
+    content = models.TextField()
+    created_time = models.DateTimeField(auto_now_add=True)
```

Comment:
Just for the future: make fields more self-descriptive because me, as a developer, who is not familiar with your code, it'll be confusing when I generate response and see that it returns date and time instead of just time. So in the future add _datetime so that it's clear that it contains date as well

Reply:
Yes, it's good point, but I am not sure I need do it specificly in this case because in readme file I have created_time field, so then I would need to change test files